_guides/trash.md

```diff
@@ -1,20 +1,20 @@
 ---
 title:  "Trash"
 tags: [content, content repository, trash, purge, delete]
-description: Trash Bin is a temporary storage for content that have been deleted. It is a central place for deleted documents where you can browse, restore or delete content permanently.
+description: Trash bin is a temporary storage for contents that have been deleted. It is a central place for deleted documents, where you can browse, restore or delete content permanently.
```

Comment:
```suggestion
description: Trash bin is a temporary storage for content that have been deleted. It is a central place for deleted documents, where you can browse, restore or delete content permanently.
```

Thank you for the fixes. Please resolve this request and we will merge your changes.

Reply:
There would be "contents" in place of "content"